common/defaults/project/catalog-repositories/dkp-catalog-applications.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: GitRepository
+metadata:
+  name: dkp-catalog-applications
+  namespace: ${releaseNamespace}
+  labels:
+    kommander.d2iq.io/gitrepository-type: catalog
+    kommander.d2iq.io/gitapps-gitrepository-type: dkp
+spec:
+  gitImplementation: go-git
+  interval: 1m0s
+  ref:
+    branch: main
+  timeout: 20s
+  url: https://github.com/mesosphere/dkp-catalog-applications
```

Comment:
for airgapped how would we point this to gitea server?

Reply:
That is up for discussion, but the CLI (during bootstrap) will do that - https://docs.google.com/document/d/18vzWTmr3O7QNvFxbWAgwusMdT2kQ3ZVFE0YZLxT0Kc4/edit#heading=h.g6xak56mm4x3

Reply:
Do we need to hold this PR until this is decided for good? I'm afraid this would break any air gap testing otherwise.

Reply:
This will not break airgapped testing. flux will just fail to reconcile the repo thus behaving as the repo is non existent.
(in our current airgapped ci we do not perform any kuttl tests and only check if kommander is installable and this pr doesnt affect any of that)

Reply:
> flux will just fail to reconcile the repo

from my point of view this is a problem and something customers will run into if not tackled. IIUC this plainly does not work in air gapped environments then?

Reply:
restructured things a bit and moved the repository definitions to installer file. updated strucuture https://github.com/mesosphere/kommander-cli/pull/335#issue-1113034744 keeps things more obvious and it is upto the user to switch the `github.com/m/dkp-catalog-application` urls to local resources when installting kommander.